### PR TITLE
This actually seems to work for fixing #76

### PR DESCRIPTION
**Pull request by godfoder**
I was poking around in the code after commenting on issue #76 and I decided to just patch in the solutions linked in the stack overflow to see if they worked.

They do indeed seem to make the dialog box reliably appear above the browser window in my use case, fixing my problem. I'm on linux right now, so I don't know for certain if this is a cross platform fix, but I'll try it out in my other environments tomorrow.

### CODE REVIEW

**Comment by chrootLogin** [comment]
@robertlugg Could you please merge this pull-request and make a new release? I need this.

File: easygui/boxes/diropen_box.py [hunk 1]
```diff
@@ -43,6 +43,9 @@ def diropenbox(msg=None, title=None, default=None):
     title = ut.getFileDialogTitle(msg, title)
     localRoot = tk.Tk()
     localRoot.withdraw()
+    localRoot.lift()
```
Comment (estasney) [inline easygui/boxes/diropen_box.py]: This solved my issue on Windows 10. It *also* solved the same issue with:

1. fileopen_box
2. filesave_box

**Comment by robertlugg** [comment]
Thank you and I'm sorry for the very long delay.

**Comment by shaozelong** [comment]
Thank you

**Pull request merged**